src/Control/Controller.php

```diff
@@ -533,8 +533,7 @@ public function render($params = null)
      * called before Controller::init(). That is, you must call it in your controller's init method
      * before it calls parent::init().
      *
-     * @deprecated 4.1.0:5.0.0 Add this controller's url to
-     * SilverStripe\Security\BasicAuthMiddleware.URLPatterns injected property instead of setting false
+     * @deprecated 4.1.0 Add this controller's url to SilverStripe\Security\BasicAuthMiddleware.URLPatterns injected property instead of setting false
```

Comment:
```suggestion
     * @deprecated 4.1.0 Add this controller's url to SilverStripe\Security\BasicAuthMiddleware.URLPatterns injected property instead
```
The "setting false" part is confusing, since this method doesn't take any arguments which would be set - this is also consistent with the message in the notice below.

Reply:
Also should the notice below have `4.1.0` as its version argument?

Reply:
The message part of this review comment hasn't been actioned?

Reply:
Updated